content/en/blog/_posts/2022-04-19-csi-nodeexpandsecret.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Use secrets while expanding csi volumes on Node"
```

Comment:
early nit:
```suggestion
title: "Kubernetes 1.24: Use Secrets While Expanding CSI Volumes On Nodes"
```

Maybe there is a better title? I might suggest a new one once I've seen what the article is introducing.

Reply:
@sftim sure, I was also in a  dilemma about the subject, so inputs are welcome!  :+1: